topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -222,20 +219,20 @@ This file contains only two columns. The first contains the exon IDs, and the se
 > ### {% icon question %} Question
 > How many exons are there in total in your file?
 >
-> *Hint: Each line now represents a different exon, so you can see the answer to this when you expand the history item, as in the image above*.
+> *Hint: Each line now represents a different exon, so you can see the answer to this when you expand the history item, as in the image below*.
 {: .question}
```

Comment:
Do you have the solution for this question?

Reply:
When I ran today it was 3,884, but I'm not sure it makes sense to put the number there, it's going to be outdated soon since UCSC SNPs are constantly updated.
I can change the hint to a solution and mention that the exact number may vary, what do you think?

Reply:
I did not realize that the value could change